cryptofeed/exchanges/bitmex.py

```diff
@@ -804,3 +810,21 @@ async def _authenticate(self, conn: AsyncConnection):
             msg = f'GET/realtime{expires}'.encode('utf-8')
             signature = hmac.new(self.key_secret.encode('utf-8'), msg, digestmod=hashlib.sha256).hexdigest()
             await conn.write(json.dumps({'op': 'authKeyExpires', 'args': [self.key_id, expires, signature]}))
+
+    async def book_callback(self, book_type: str, book: OrderBook, receipt_timestamp: float, timestamp=None, raw=None,
```

Comment:
I dont understand the purpose of overloading this method in bitmex. if you need to do something special for timestamps, why not just do that in the method that parses the book?

Reply:
Sorry, you're right. It is more direct to have this change in book parser method, `_book`.
I did this as I am having this fix in my project. Overriding `book_callback` will be a shorter solution.

Let me update this change.